fix k8s exporter configuration changes on docs (#2682)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/kubernetes-stack/kubernetes/advanced.md

</TabItem>

<TabItem value="logging">

The `loggingLevel` parameter is used to control the logging level of logs in K8s exporter's pod.

- **Default value**: `info` (show all info and error logs, including info logs of successful updates)
- **Use case**: Set the value to `error`, if you want to clear out info logs. Error logs will be reported.

</TabItem>

<TabItem value="metrics">

The `metricsEnabled` parameter is used to control if the k8s exporter would expose a `/metrics` endpoint compatible with OTLP.

- **Default value**: `true`

The `metricsPort` parameter is used to control the port number of the metrics server.

- **Default value**: `9090`
- **Use case**: Set the value to other port if from some reason you cannot expose the metrics server on `9090`.

</TabItem>

<TabItem value="bulkSync">

> The `bulkSyncMaxPayloadBytes` parameter is used to control the size (in bytes) of the upsert bulks to port.
> 
> - **Default value**: `1048576`
> 
> The `bulkSyncMaxEntitiesPerBatch` parameter is used to control the number of entities to be sent as part of an upsert bulk to port.

Reviewer: ```suggestion
The `bulkSyncMaxPayloadBytes` parameter is used to control the size (in bytes) of the bulk upserts to Port.

- **Default value**: `1048576`

The `bulkSyncMaxEntitiesPerBatch` parameter is used to control the number of entities to be sent as part of a bulk upsert to Port.
```

- **Default value**: `20`

> The `bulkSyncBatchTimeoutSeconds` parameter is used to control the time (in seconds) to wait until flushing an upsert bulk to port when the max entities number or size not reached.

Reviewer: ```suggestion
The `bulkSyncBatchTimeoutSeconds` parameter is used to control the time (in seconds) to wait until flushing a bulk upsert to Port (even if the maximum number/size of entities has not been reached).
```
